script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -528,13 +528,13 @@ What makes an Ancestor an Ancestor is the character of their blood."
 [sha:9a97080f59c6c85b4a94a31acf9dc537cf5801dd]{
 -- Page 609, Offset 18105.
 -- それぞれ戴いた真理、渇きの根底となる世界観。
-The truths they have each been crowned;
- The worldviews that underpin their thirst;
+the truths they have each been crowned;
+ the worldviews that underpin their thirst;
 }
 [sha:ded30ad443ac5e43d2d483b697e500c9c52eb5ad]{
 -- Page 609, Offset 18106.
 -- その血を巡らせるだけで惑星の物理法則を塗り替える特異点。
-The singularities that paint over the planet's physical laws just by circulating their blood.
+the singularities that paint over the planet's physical laws just by circulating their blood.
```

Comment:
I prefer to keep all new lines starting with caps tbh

Reply:
![image](https://user-images.githubusercontent.com/90473809/179356964-501c19e2-90d3-49a9-a47b-9d5ed79a97ae.png)
looks like this in game
should i bring down the "The worldviews" one so all of them start in a new line?